meson.build

```diff
@@ -2031,6 +2031,7 @@ make_directive_index_py = find_program('tools/make-directive-index.py')
 sync_docs_py = find_program('tools/sync-docs.py')
 make_man_index_py = find_program('tools/make-man-index.py')
 meson_render_jinja2 = find_program('tools/meson-render-jinja2.py')
+ukify_py = find_program('src/ukify/ukify.py')
```

Comment:
Please move this line to src/boot/meson.build in the `if conf.get('ENABLE_UKIFY') == 1 ...` block.